bika/lims/browser/aggregatedanalyses/aggregatedanalyses.py

```diff
@@ -58,15 +58,36 @@ def __init__(self, context, request, **kwargs):
             }
         self.review_states = [
             {'id': 'default',
-             'title':  _('Results pending'),
+             'title': _('Assignment pending'),
+             'transitions': [{'id': 'submit'},
+                             {'id': 'cancel'},
+                             # {'id': 'assign'}
+                             ],
+             'contentFilter': {
+                 'review_state': ['sample_received'],
+                 'cancellation_state': 'active', },
+             'columns': ['AnalysisRequest',
+                         'Worksheet',
+                         'Service',
+                         'Result',
+                         'Uncertainty',
+                         'Partition',
+                         'Method',
+                         'Instrument',
+                         'Analyst',
+                         'state_title',
+                         ]
+             },
```

Comment:
Since the user can set results directly to this view (especially when the state is `sample_received`), at lease, add the following columns, keeping the same order as in AnalysesView: 

DetectionLimit
https://github.com/senaite/senaite.core/blob/master/bika/lims/browser/analyses.py#L109-L112

Specifications:
https://github.com/senaite/senaite.core/blob/master/bika/lims/browser/analyses.py#L118-L120